charts/redisoperator/Chart.yaml

```diff
@@ -1,4 +1,15 @@
+annotations:
+  category: Redis Operator
+appVersion: 3.1.0
 apiVersion: v1
 description: A Helm chart for the Spotahome Redis Operator
-name: redisoperator
-version: 3.1.0
+name: redis-operator
+version: 3.1.2
```

Comment:
I would update the version of the chart to 4.0.0 since it includes breaking changes like modifications on immutable fields like deployment selector

Reply:
```suggestion
version: 4.0.0
```